### DISCUSSION TITLE
GUI update companion breaks SSL?

### DISCUSSION THREAD

**Discussion by docdoceg** [requarks/wiki#5994] [Help / Questions]
OS: Ubuntu 22.04 LTS and Docker
Wiki.js version: 2.5.295
Database engine: Postgres 11

Hello, after running the update companion to 2.5.295, connections through https were refused and a manual "stop, rm, create, start" had to be done with the LetsEncrypt variables for the wiki to be accessible again. Is this a limitation with the update companion? It seems to wipe any SSL configuration.

All containers were made about a month ago, so I'm assumed the [Unable to upgrade from the Administration Area](https://docs.requarks.io/en/troubleshooting#unable-to-upgrade-from-the-administration-area) issue isn't the cause because it did actually update, just broke SSL.

Thanks for any clarification

**Comment by NGPixel** [comment]
The update companion simply pull the latest image and recreates the container using the same configuration (env vars, name, mounts, etc.) as it was when running.
